x/stakeibc/keeper/unbonding_records_sweep_unbonded_tokens_test.go

```diff
@@ -143,62 +143,58 @@ func (s *KeeperTestSuite) SetupSweepUnbondedTokens() SweepUnbondedTokensTestCase
 func (s *KeeperTestSuite) TestSweepUnbondedTokens_Successful() {
 	tc := s.SetupSweepUnbondedTokens()
 	_ = tc
-	success, successful_sweeps, sweep_amounts, failed_sweeps := s.App.StakeibcKeeper.SweepAllUnbondedTokens(s.Ctx())
-	s.Require().Equal(success, true, "sweep all tokens succeeds")
-	s.Require().Equal(len(successful_sweeps), 2, "sweep all tokens succeeds for 2 host zones")
-	s.Require().Equal(len(sweep_amounts), 2, "sweep all tokens succeeds for 2 host zones")
-	s.Require().Equal(len(failed_sweeps), 0, "sweep all tokens fails for no host zone")
-	s.Require().Equal(sweep_amounts, []int64{2_000_000, 3_000_000}, "correct amount of tokens swept for each host zone")
+	success, successfulSweeps, sweepAmounts, failedSweeps := s.App.StakeibcKeeper.SweepAllUnbondedTokens(s.Ctx())
```

Comment:
I think now that these functions have been refactored, it makes more sense to focus the unit tests on `SweepAllUnbondedTokensForHostZone`. So all you have to do is define and test state for one host zone, and it will let you get more granular with the test coverage

And then you can just have maybe two tests on `SweepAllUnbondedTokens` that test 1) multiple host zones all succeeding, and 2) multiple host zones with one failing

Reply:
agreed! although do you think we should do this now or after launch?

Reply:
You'll probably hate this answer, but I'd argue that it's a high priority refactor since it gives a lot more coverage (just from scrolling, it looks like there's at least 4 branches that aren't traversed). That said, I think it's best done in a separate PR (since it's a pretty involved refactor), I don't necessarily think it needs to get done today, and I'm also happy to take it. But I'd prioritize it a little higher than our typical post-launch tech debt refactoring